maintenance/image-options/private-registry/private-registry-archive.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Install from a private registry
+description: Install and configure Calico Enterprise in a private registry. 
+layout: null
+---
+
+[//]: #  This is used in manifests archive, though it doesn't appear on the website
+
+{% assign operator = site.data.versions.first.tigera-operator %}
+
+### Before you begin...
+
+- Configure pull access to your private registry
+- [Configure pull access to Tigera's private container registry]({{ "/getting-started/calico-enterprise#get-private-registry-credentials-and-license-key" | absolute_url }} ).
+
+
+{% include content/private-registry-regular.md %}
```

Comment:
What's the reason for doing this as an include rather than inline content?

Reply:
I'd say the reason for the include is so it can be used here and in the `maintenance/image-options/private-registry/private-registry-regular.md`, I'm not sure about the need for this archive specific doc though.